lib/v2/uber/radar.js

```diff
@@ -0,0 +1,13 @@
+module.exports = {
+    'www.uber.com': {
```

Comment:
You need to specify the root domain only.
```suggestion
    'uber.com': {
```